modules/nw-multus-tap-object.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+[id="nw-multus-tap-object_{context}"]
+= Configuration for a TAP additional network
+
+The following object describes the configuration parameters for the TAP CNI
+plugin:
+
+.TAP CNI plugin JSON configuration object
+[cols=".^2,.^2,.^6",options="header"]
+|====
+|Field|Type|Description
+
+|`cniVersion`
+|`string`
+|The CNI specification version. The `0.3.1` value is required.
+
+|`name`
+|`string` (required)
+|The value for the `name` parameter you provided previously for the CNO configuration.
+
+|`type`
+|`string` (required)
+|The name of the CNI plugin to configure: `ipvlan`.
+
+|`mac`
+|`string` (optional)
+|Request the specified MAC address for the interface.
+
+|`mtu`
+|`integer`
+|Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
+
+|`selinuxcontext`
+|`string` (optional)
+|For systems with the selinux security module enabled, the context under which to create the tap device.
```

Comment:
This is a copy and paste from the upstream documentation. We are a system with selinux. Please rephrase this.